userspace/falco/configuration.cpp

```diff
@@ -177,6 +177,22 @@ void falco_configuration::load_yaml(const std::string& config_name, const yaml_h
 		user_agent = config.get_scalar<std::string>("http_output.user_agent","falcosecurity/falco");
 		http_output.options["user_agent"] = user_agent;
 
+		bool insecure;
+		insecure = config.get_scalar<bool>("http_output.insecure", false);
+		http_output.options["insecure"] = insecure? std::string("True") : std::string("False");
```

Comment:
Isn't there a pair of string <-> bool primitives that we can reuse to avoid custom formats? If not, can we at least make this case-insensitive (or tied to lower case)?

Reply:
The `config` type for `outputs` uses a` map<string, string>` flexible enough to hold arbitrary values as long as we represent them as strings. https://github.com/falcosecurity/falco/blob/4836978ba9c054cfb752616a252c8d2bda8f1870/userspace/falco/outputs.h#L34-L38
Also the string representation of the `bool` type is internal, we are still requiring the users to use a `bool` value in the configuration. 

I didn't find a more elegant/appropriate way to pass the `bool` value to the `http_output` object. If we stick with this solution we should make the string values lowercase.

Reply:
Let's take the simple route and just make it lowercase then.